README.md

```diff
@@ -1 +1,163 @@
-# authorino-operator
\ No newline at end of file
+# Authorino Operator
+
+[![License](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0)
+
+## Overview
+
+A Kubernetes Operator to manage [Authorino](https://github.com/Kuadrant/authorino) deployments.
+
+## Custom Resources
+
+* [Authorino](https://github.com/Kuadrant/authorino-operator/blob/3592a17868250de7079f26584059d09bbb51ff70/config/crd/bases/operator.authorino.kuadrant.io_authorinos.yaml), Resposible for install, manager and configure Authorino, each CRD represents an instance of Authorino running on the cluster.
+
+### Authorino's API Specification
+
+#### Spec
+
+| Field | json field | Type | Info |
```

Comment:
Perhaps we could go

```suggestion
| Field | Type | Description |
```

In "Field", we put the field name as in the spec (e.g. `logLevel`).

In "Type", pretty much what you proposed, except that instead of "object", we could use the name of the type you specified below, with a link to the section. E.g. 
```
| Field | Type | Description |
| `listener` | [Listener](#listener) | Specification of the authorization service (gRPC interface) |
```

(Just a suggestion)

Reply:
Sure, good idea I don't think the field name that matches with the API is too important